jenkins-master/jobs/scripts/workspace/virtualenv/.gitignore

```diff
@@ -0,0 +1,10 @@
+virtualenv.egg-info
```

Comment:
Maybe I should add it as zip and let the environment.py script unzip it. What do you think?